opensrp-reveal/src/main/java/org/smartregister/reveal/util/CardDetailsUtil.java

```diff
@@ -54,6 +54,9 @@ public static void formatCardDetails(CardDetails cardDetails) {
                 break;
             case BusinessStatus.SPRAYED:
             case BusinessStatus.COMPLETE:
+                cardDetails.setStatusColor(R.color.partially_sprayed);
```

Comment:
I didn't change this below to ensure backwards compatibility. I didn't know if we may get an event from a previous year or testing on Reveal stage that would cause the app to crash if it wasn't there.